demo/src/public/versions.json

```diff
@@ -0,0 +1 @@
+["12.x"]
```

Comment:
Hmm, I think I'd like to put all data like text and URLs here as well. So in case things change, we won't need to rebuild older versions.